src/Test/Perf/util/TraceManager.cs

```diff
@@ -63,24 +59,24 @@ public void Initialize()
 
         public void Setup()
         {
-            ShellOutVital(_cpcPath, "/Setup /DisableArchive", _verbose, _logger);
+            ShellOutVital(_cpcPath, "/Setup /DisableArchive", _verbose, _logger, workingDirectory: "");
```

Comment:
If we don't specify the value explicitly then we will try to run the command inside `MyWorkingDirectory()` directory. This scenario holds true only when called from csx files. 

In short, both calling `InitUtilities()` from within library and not setting the value of _workingDirectory are wrong. The correct approach is to always provide the directory to run the command if it is run outside of the context of csx.